### ISSUE TITLE
No output when source has no audio

### ISSUE THREAD

**Issue by Draghmar** [moisesmcardona/rav1e_gui#12]
Hi
I've noticed that when source file does not have an audio the encoding goes all the way but there's no output file. The same file with any audio track is encoded like it should.
I'm guessing it's because there's no rav1e-audio.opus file and merging silently fails.
The audio track is removed with ffmpeg -an switch.

BTW it would be great to have build in checkbox to remove audio right from the GUI. :)
```
2019-06-03 11:55:52 || y4m-part-000001.y4m || Key Frames:      2    avg size:   93945 B
2019-06-03 11:55:52 || y4m-part-000001.y4m || Inter:         248    avg size:    2545 B
2019-06-03 11:55:52 || y4m-part-000001.y4m || Intra Only:      0    avg size:       0 B
2019-06-03 11:55:52 || y4m-part-000001.y4m || Switch:          0    avg size:       0 B
2019-06-03 11:55:52 || Video part y4m-part-000001.y4m Encoding complete.
2019-06-03 11:55:52 || Video Segments Encoded
2019-06-03 11:55:52 || Concatenating encoded video segments
2019-06-03 11:55:52 || Video concatenated
2019-06-03 11:55:52 || Merging audio and video files
2019-06-03 11:55:53 || Merge complete
```

**Comment by moisespr123** [comment]
Hi,

This is because the merging is actually wanting an audio file. I'll fix this in the next version. Thanks for reporting!

**Comment by moisespr123** [comment]
Fixed in v1.14.
